actionlint/action.yml

```diff
@@ -0,0 +1,89 @@
+---
+name: Action lint
+
+description: Linting for GitHub workflows
+
+inputs:
+  version:
+    description: Actionlint version
+    default: "1.7.3"
+  ignore:
+    description: Multiline; Ignore errors by messaging using regular expression.
+  format:
+    description: Format error messages using Go template
+  custom-config-file:
+    description: Actionlint custom configuration file path within original repo
+  sanitize-runner-labels:
+    description: Sanitize custom runner labels to only those supported by infra-team
+    default: "true"
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v4
+
+    - run: echo "::add-matcher::${{ github.action_path }}/actionlint-matcher.json"
+      shell: bash
+
+    - name: Download actionlint
+      id: get_actionlint
+      env:
+        ACTIONLINT_VERSION: ${{ inputs.version }}
+      run: bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash) $ACTIONLINT_VERSION
+      shell: bash
+
+    - name: Check workflow files
+      env:
+        FORMAT: ${{ inputs.format }}
+        IGNORE_MESSAGES: ${{ inputs.ignore }}
+        CUSTOM_CONFIG_FILE: ${{ inputs.custom-config-file }}
+        DEFAULT_CONFIG_FILE: ${{ github.action_path }}/actionlint.yaml
+        SANITIZE_CUSTOM_LABELS: ${{ inputs.sanitize-runner-labels }}
+      run: |
+
```

Comment:
Do you think it would make sense to move this entire shell block to a separate script like you already did with the santize script?

Reply:
Not so sure about this right now. I would prefer to keep it this way for now